actions/docker-build/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Publish Docker image"
+description: "Use a Dockerfile to publish image to any host"
```

Comment:
Please change

Reply:
Please also change the name

Reply:
This is not resolved. The name is still `Publish Docker image`, but this action is not publishing anything